dsp-shared/src/main/scala/dsp/valueobjects/Project.scala

```diff
@@ -171,15 +180,15 @@ object Project {
 
 object ProjectErrorMessages {
   val ShortcodeMissing           = "Shortcode cannot be empty."
-  val ShortcodeInvalid           = "Shortcode is invalid."
-  val ShortnameMissing           = "Shortname cannot be empty."
-  val ShortnameInvalid           = "Shortname is invalid."
-  val LongnameMissing            = "Longname cannot be empty."
-  val LongnameInvalid            = "Longname is invalid."
+  val ShortcodeInvalid           = (v: String) => s"Shortcode is invalid: $v"
```

Comment:
The spelling of Shortcode is inconsistent. Could you check and unify it? I think it would be good to also check Shortname and Longname

Reply:
I unified it a bit... but it's not perfect yet. I must admit that I don't really like the camel case spelling, especially in readable strings (spec descriptions, error messages, comments, etc.). And I can't go ham on it, without touching V2 code, which I don't want at this stage.

Regarding Longname: That one is now "Name" which should be consistent everywhere.  
Regarding Shortname: I did not include that in the project yet (needs domain model first, as I'm not sure we're keeping this at all), so I left it unchanged for now.  
I understand this creates some inconsistencies, but for now I think it should be acceptable. I hope that's ok.

Reply:
Sure, it's just suggestions and it doesn't matter too much for the current slices as they are only for trying out ZIO.

Reply:
True - but at the same time I don't want to override those suggestions without explaining my reasoning, if that means that other people then have to live with a messy codebase ;-)

Reply:
Sure, I understand! :)

Reply:
Well, this has now been changed. But now we have `ShortCode` vs. `Shortname` which is inconsistent again